api/nakadi-event-bus-api.yaml

```diff
@@ -42,18 +42,18 @@ info:
     Except for defined enrichment rules, Nakadi will never manipulate the content of any Event.
 
     Clients of Nakadi can be grouped in 2 categories: **EventType owners** and **Clients** (clients in turn
-    are both **Producers** and **Consumers** of Events). Application owners interact with Nakadi via the
+    are both **Producers** and **Consumers** of Events). Event Type owners interact with Nakadi via the
     **Schema Registry API** for the definition of EventTypes, while Clients via the streaming API for
     submission and reception of Events.
 
     A low level **Unmanaged API** is available, providing full control and responsibility of position
     tracking and partitioning (and therefore ordering) to the Clients.
 
-    In the high level **Subscriptoin API** the consumption of Events proceeds via the establishment of a named
-    **Subscription** to an EventType. Subscriptions are persistent relationships from an Application (which
-    might have several instances) and the stream of one or more EventType's, whose consumption tracking is
-    managed by Nakadi, freeing Consumers from any responsibility in tracking of the current position on
-    a Stream.
+    In the high level **Subscription API** the consumption of Events proceeds via the establishment of a
```

Comment:
Is the official naming now "Subscription API" and "Low Level API"?
I used "managed" and "unmanaged" API the past few days. Have we agreed on something here?